oblib/taxonomy_semantic.py

```diff
@@ -280,18 +280,18 @@ def validate_ep(self, data):
             return False
 
     def concepts_ep(self, data):
-        """Return a list of all concepts in an end point."""
+        """Return a list of all concepts in an entry point."""
         if data in self._concepts:
```

Comment:
Would help to state the expected type for `data`

Reply:
I'll add this to the api signature review session on Tuesday.  We probably should take the PDF available on readthedocs and review it without the code with an eye on "could someone write code simply via readthedocs".  I'll keep #38 (docstrings) open for now.

Reply:
Works for me, Its certainly a larger issue that this PR.